pages/common/pytest.md

```diff
@@ -0,0 +1,28 @@
+# pytest
+
+> Run Python tests.
+> More information: <https://docs.pytest.org/>.
+
+- Run tests from a specific file:
+
+`pytest {{path/to/test_file.py}}`
+
+- Run all tests with a name that matches a given substring:
+
+`pytest -k {{substring}}`
+
+- Stop after the first test failure:
+
+`pytest --exitfirst`
+
+- Run tests matching or excluding markers:
+
+`pytest -m {{marker_name1 and not marker_name2}}`
+
+- Run until a test failure, continuing from the last failing test:
+
+`pytest --stepwise`
+
+- Run tests without capturing output:
+
+`pytest --capture=no`
```

Comment:
```suggestion
```

Optional fix: `--capture` is not mentioned in a [man](https://manned.org/pytest.1) page.
Note: possible bug in the corresponding man page.

Reply:
Usually the option is just `-s`, but I was following the style guide to use the long option.